locale/ru_RU/emailTemplates.xml

```diff
@@ -7,720 +7,676 @@
 <br />
-Если хотите прекратить получение извещений, пожалуйста перейдите по ссылке  {$unsubscribeLink} и введите свой почтовый адрес и пароль.<br />
+Пожалуйста, подтвердите, что Вы сможете сделать этот важный вклад в работу нашего журнала. Жду вашего ответа.<br />
 <br />
-{$principalContactSignature}]]></body>
+{$editorialContactSignature}]]></body>
+		<description>Это письмо отправляется автоматически, когда дата предоставления рецензии рецензентом прошла (смотрите Параметры рецензирования в Настройки &gt; Рабочий процесс &gt; Рецензирование) и прямой доступ рецензента по ссылке отключен. Запланированные задачи должны быть включены и настроены (смотрите файл конфигурации сайта).</description>
 	</email_text>
-	<email_text key="NOTIFICATION_MAILLIST_WELCOME">
-		<subject>Добро пожаловать в список рассылки {$siteTitle} !</subject>
-		<body><![CDATA[Вы подписались на получение извещений с сайта {$siteTitle}.<br />
+	<email_text key="REVIEW_REMIND_AUTO_ONECLICK">
+		<subject>Автоматическое напоминание о рецензии на материал</subject>
+		<body><![CDATA[{$reviewerName}!<br />
 <br />
-Пожалуйста нажмите на ссылку для подтверждения своего запроса и добавления Вашего почтового адреса в список рассылки: {$confirmLink}<br />
+Это напоминание о нашем запросе Вашей рецензии на материал «{$submissionTitle}» для журнала «{$contextName}». Мы надеялись получить эту рецензию до {$reviewDueDate}, и это письмо было автоматически сгенерировано и отправлено, так как эта дата уже прошла. Мы будем рады, если Вы как можно скорее ее подготовите.<br />
 <br />
-Если не хотите получать извещения, пожалуйста нажмите на ссылку {$unsubscribeLink}  и введите свой почтовый адрес и пароль.<br />
+URL материала: <a href="{$submissionReviewUrl}">{$submissionReviewUrl}</a><br />
 <br />
-Пароль для отключения извещений: {$password}<br />
+Пожалуйста, подтвердите, что Вы сможете сделать этот важный вклад в работу нашего журнала. Жду вашего ответа.<br />
 <br />
-{$principalContactSignature}]]></body>
+{$editorialContactSignature}]]></body>
+		<description>Это письмо отправляется автоматически, когда дата предоставления рецензии рецензентом прошла (смотрите Параметры рецензирования в Настройки &gt; Рабочий процесс &gt; Рецензирование) и прямой доступ рецензента по ссылке включен. Запланированные задачи должны быть включены и настроены (смотрите файл конфигурации сайта).</description>
 	</email_text>
-	<email_text key="USER_VALIDATE">
-		<subject>Подтвердите свой электронный адрес</subject>
-		<body><![CDATA[{$userFullName}<br />
-<br />
-Вы зарегистрировались в журнале {$contextName}, но перед тем как начать пользоваться своей учетной записью, нужно подтвердить правильность Вашего электронного адреса. Чтобы это сделать просто пройдите по ссылке:<br />
+	<email_text key="EDITOR_DECISION_ACCEPT">
+		<subject>Решение редакции</subject>
+		<body><![CDATA[{$authorName}!<br />
 <br />
-<a href="{$activateUrl}">{$activateUrl}</a><br />
+Мы приняли решение относительно Вашего мат
```

Comment:
Well, @mtub, I think maybe we can fix it in the following way: "Здравствуйте, {$userFullName}!". It's also not very good, because when you say "Hello", you usually say "Hello, Peter" or "Hello, mister Petersfield", but  not "Hello, Petersfield" or "Hello, Peter Petersfield" (and "Здравствуйте, {$userFullName}!" looks like "Hello, Peter Petersfield!"). But it could be better than just "{$userFullName}!" (i.e. "Peter Petersfield!")